### PR TITLE
creating harness tests for ImageBitmap

### PR DESCRIPTION
**Pull request by xidachen**
At this moment, web platform tests doesn't have enough tests to cover the case of ImageBitmap. This will be a starting point of testing ImageBitmap. More will be added later on.

Here we tests createImageBitmap from 6 different kinds of sources: HTMLImageElement, HTMLCanvasElement, HTMLVideoElement, ImageData, Blob, ImageBitmap without any crop.

To ensure the correctness, I used latest chromium build and firefox nightly build, and all tests passed.

Please review.

### CODE REVIEW

**Comment by Ms2ger** [comment]
Revert the submodule update, please.

**Comment by xidachen** [comment]
@Ms2ger I have reverted the submodule update, PTAL.

File: 2dcontext/imagebitmap/common.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+function testDrawImageBitmap(source)
+{
+    var canvas = document.createElement("canvas");
+    canvas.width = 20;
+    canvas.height = 20;
+    var ctx = canvas.getContext("2d");
+    ctx.clearRect(0, 0, canvas.width, canvas.height);
+    return createImageBitmap(source).then(imageBitmap => {
```
Comment (jdm) [inline 2dcontext/imagebitmap/common.js]: We try to avoid ES6 features like this in these tests.

**Comment by jdm** [comment]
This looks good apart from the one nit.

File: 2dcontext/imagebitmap/common.js (hunk 1 above)
Comment (jgraham) [inline 2dcontext/imagebitmap/common.js]: (because they don't yet work in the latest shipping versions of all relevant UAs)

**Comment by annevk** [comment]
Since array functions are now widely supported and used throughout web-platform-tests I've taken the liberty to merge this. Thanks @xidachen!

**Pull request merged**